libavogadro/src/extensions/crystallography/crystallographyextension.cpp

```diff
@@ -2191,6 +2195,13 @@ namespace Avogadro
     CE_CACTION_DEBUG(PrimitiveReduceIndex);
     CE_CACTION_ASSERT(PrimitiveReduceIndex);
 
+    // PrimitiveReduceStandardIndex,
+    a = new QAction(tr("Reduce Cell (&Primitive and Standardize)"), this);
```

Comment:
Yeah we can. Technically, there is a "Niggli reduction" as well, so to make sure don't confuse the two types of reduction, I think we need the word "Primitive"

Is "Primitive Reduce and Standardize" too long? Or will that work?